src/services/SQSService.js

```diff
@@ -49,13 +51,16 @@ class SQSService {
         .findOne({ _id: mongoose.Types.ObjectId(type.template_id) })
         .exec();
 
-      const generatedTemplate = Mustache.render(template, { message: JSON.parse(event.Records[0].body).text });
-      const now = new Date();
+      const generatedTemplate = Mustache.render(template.template, { message: JSON.parse(event.Records[0].body).text });
+      const message = {
+        subject: template.subject,
```

Comment:
This should be generated using mustache.render() just like the message body is generated. Maybe this dynamic generation is in a separate pull request?

Reply:
it's also already done in email templates pr's